frontend/src/pages/instance/Settings/Editor.vue

```diff
@@ -137,7 +137,7 @@ export default {
             })
             await InstanceApi.updateInstance(this.project.id, { settings })
             this.$emit('instance-updated')
-            alerts.emit('Instance successfully updated.', 'confirmation')
+            alerts.emit('Instance settings successfully updated. NOTE: changes will be applied once the instance is restarted.', 'confirmation', 6000)
```

Comment:
```suggestion
            alerts.emit('Instance settings successfully updated. NOTE: changes will be applied once the instance is restarted.', 'confirmation', 6000)
```

Tbhis implies it may restart automatically, I think a more explicit CTA is required: "Please restart your instance for these changes to take effect"?

Reply:
Good point. 

will reword